client/tooltips.js

```diff
@@ -0,0 +1,156 @@
+module.exports = {
+  speciesEditor: {
+    name: "Names for species, parameters, reactions, events, and rules must be unique.",
+
+    initialValue: "Initial population of a species.",
+
+    annotation: "An optional note about the species.",
+
+    remove: "A species may only be removed if it is not a part of any reaction, event assignment, or rule.",
+    
+    speciesMode: "Concentration - Species will only be represented as deterministic.<br>Population - Species will only be represented as stochastic.<br>Hybrid "+
+        "Concentration/Population - Allows a species to be represented as either deterministic or stochastic. This allow you to customize the mode of individual species "+
+        "and set the switching tolerance or minimum value for switching.",
+
+    mode: "Concentration - Species will only be represented as deterministic.<br>Population - Species will only be represented as stochastic.<br>Hybrid "+
+        "Concentration/Population - Allows a species to be represented deterministically and/or stochastically.",
+
+    switchValue: "Switching Tolerance - Tolerance level for considering a dynamic species deterministically, value is compared to an estimated sd/mean population of a "+
+        "species after a given time step. This value will be used if a switch_min is not provided.<br>Minimum Value For Switching - Minimum population value at which "+
+        "species will be represented as Concentration."
+  },
+  parametersEditor: {
+    name: "Names for species, parameters, reactions, events, and rules must be unique.",
+    
+    expression: "A parameter value or a mathematical expression calculating the parameter value.",
+    
+    annotation: "An optional note about a parameter.",
+    
+    remove: "A parameter may only be removed if it is not used in any reaction, event assignment, or rule."
+  },
+  reactionsEditor: {
+    name: "Names for species, parameters, reactions, events, and rules must be unique.",
+    
+    annotation: "An optional note about the reaction.",
+    
+    rate: "The rate of the mass-action reaction.",
+    
+    propensity: "The custom propensity expression for the reaction.",
+    
+    reactant: "The reactants that are consumed in the reaction, with stoichiometry.",
+    
+    product: "The species that are created by the reaction event, with stoichiometry.",
+    
+    reaction: "For a species that is NOT consumed in the reaction but is part of a massaction reaction, add it as both a reactant and a product.\nMass-action reactions "+
```

Comment:
This is a weird place to put this note.

Reply:
This does feel more of an instruction than a definition.  Maybe this should be in the quickstart/tutorial?